nerfstudio/viewer/app/src/modules/ViewerWindow/ViewerWindow.jsx

```diff
@@ -313,6 +313,49 @@ export default function ViewerWindow(props) {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [isWebsocketConnected]);
 
+  const throttledClickSender = makeThrottledMessageSender(
+    viser_websocket,
+    10,
+  );
+  useEffect(() => {
+    const onMouseDouble = (e) => {
+      const BANNER_HEIGHT = 50;
```

Comment:
Not sure how, but this should be a same value here and in `scene.jsx`. Could cause bugs in the future if one is updated but not the other.

Reply:
maybe `myRef.current.offsetTop` ? we should probably also name this something more meaningful

Reply:
I agree that `BANNER_HEIGHT` should be synced between the two files.
Seems like `myRef.current.offsetTop` returns 0 right now though (instead of expected 50), this might need some more investigation...